reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -425,10 +425,12 @@ def initialize_traefik(self):
         try:
             namespace = 'kube-system'
             label_selector = 'app=traefik'
-            cmd = ('helm install stable/traefik --namespace {} --values {} '
-                   '--name {}').format(namespace,
-                                       traefik_configuration_file_path,
-                                       traefik_release_name)
+            cmd = ('helm install {} stable/traefik --namespace {} '
```

Comment:
I am having the following problem:
```console
$ helm install reana-traefik stable/traefik \
  --namespace kube-system --values reana_cluster/configurations/helm/traefik/minikube.yaml 
Error: failed to download "stable/traefik" (hint: running `helm repo update` may help)
```
Have you seen it? (I am having the exact same versions as you have listed plus this PR and R-Commons checkout)

Reply:
Could you try to execute the following command before ``reana-cluster init ...``

`$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/`

https://helm.sh/docs/intro/quickstart/#initialize-a-helm-chart-repository